Mage.Sets/src/mage/cards/s/StreetwiseNegotiator.java

```diff
@@ -38,11 +26,11 @@ public StreetwiseNegotiator(UUID ownerId, CardSetInfo setInfo) {
 
         // Backup 1
         BackupAbility backupAbility = new BackupAbility(this, 1);
-        this.addAbility(backupAbility);
-
         // This creature assigns combat damage equal to its toughness rather than its power.
-        backupAbility.addAbility(new SimpleStaticAbility(new CombatDamageByToughnessEffect(filter, false)
-                .setText("this creature assigns combat damage equal to its toughness rather than its power")));
+
+        backupAbility.addAbility(new SimpleStaticAbility(new CombatDamageByToughnessSourceEffect(Duration.EndOfTurn)));
```

Comment:
Is it works here? If you setup `Duration.EndOfTurn` then effect must be deleted by game engine at the end of the turn. But card's ability must works until end of the game.

Reply:
Well, that backup's code looks buggy and must be checked. Some abilities added directly to the card, but another aren't (only one card uses dynamic gaining abilities: [[Conclave Sledge-Captain]])

![shot_230831_050524](https://github.com/magefree/mage/assets/8344157/1661582b-1970-4809-97e6-ca9b5c03e4c6)
![shot_230831_050503](https://github.com/magefree/mage/assets/8344157/a7bd19ac-164c-444d-b04d-200f9b6635fc)

Reply:
And second problem here (well, original problem) -- `CombatDamageByToughnessSourceEffect` must be part of the card, not backup:

![shot_230831_051725](https://github.com/magefree/mage/assets/8344157/ea1fc4a4-80e9-4457-8b6e-d130eec4c97a)

Reply:
never mind, all fine))